src/components/dropdown/dropdown.js

```diff
@@ -663,9 +680,11 @@ class Dropdown extends React.Component {
     }
 
     content.push(
-      <div { ...this.listBlockProps } ref={ (node) => { this.listBlock = node; } }>
-        { this.listHTML }
-      </div>
+      <Portal onReposition={ () => { this.calculatePosition(); } }>
```

Comment:
just noticed you don't need to create a function here...

```
<Portal onReposition={ this.calculatePosition }>
```
rather than

```
<Portal onReposition={ () => { this.calculatePosition(); } }>
```